DigitalLearningSolutions.Data.Tests/Services/UserServiceTests.cs

```diff
@@ -497,6 +497,26 @@ public void NewEmailAddressIsValid_returns_true_for_delegate_only_with_unchanged
             A.CallTo(() => userDataService.GetDelegateUsersByEmailAddress(email)).MustNotHaveHappened();
         }
 
+        [Test]
+        public void NewEmailAddressIsValid_returns_true_with_unchanged_email_with_different_case()
+        {
+            // Given
+            const string email = "email@test.com";
+            const string capsEmail = "Email@test.com";
+            var adminUser = UserTestHelper.GetDefaultAdminUser(emailAddress: email);
+            var delegateUser = UserTestHelper.GetDefaultDelegateUser(emailAddress: capsEmail);
+            A.CallTo(() => userDataService.GetAdminUserById(adminUser.Id)).Returns(adminUser);
+            A.CallTo(() => userDataService.GetDelegateUserById(delegateUser.Id)).Returns(delegateUser);
+
+            // When
+            var result = userService.NewEmailAddressIsValid(email, 7, 2, 2);

```

Comment:
Can we get these properties (except for email) from the adminUser and delegateUser objects please, just in case they change in the future.